doc/source/user_guide/io.rst

```diff
@@ -3931,15 +3890,6 @@ returned, this is equivalent to passing a
 ``start`` and ``stop`` parameters can be specified to limit the total search
 space. These are in terms of the total number of rows in a table.
 
-.. ipython:: python
-
-   # this is effectively what the storage of a Panel looks like
-   wp.to_frame()
-
-   # limiting the search
-   store.select('wp', "major_axis>20000102 & minor_axis=['A', 'B']",
-                start=0, stop=10)
```

Comment:
There is still a `store.remove('wp', 'major_axis > 20000102')` somewhere below (see the travis log output)

Reply:
```
>>>-------------------------------------------------------------------------
Exception in /home/travis/build/pandas-dev/pandas/doc/source/user_guide/io.rst at block ending on line 4230
Specify :okexcept: as an option in the ipython:: block to suppress this message
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-480-77cd4accde3f> in <module>
----> 1 store.remove('wp', 'major_axis > 20000102')
~/build/pandas-dev/pandas/pandas/io/pytables.py in remove(self, key, where, start, stop)
    901         where = _ensure_term(where, scope_level=1)
    902         try:
--> 903             s = self.get_storer(key)
    904         except KeyError:
    905             # the key is not a valid store, re-raising KeyError
~/build/pandas-dev/pandas/pandas/io/pytables.py in get_storer(self, key)
   1155         group = self.get_node(key)
   1156         if group is None:
-> 1157             raise KeyError('No object named {key} in the file'.format(key=key))
   1158 
   1159         s = self._create_storer(group)
KeyError: 'No object named wp in the file'
<<<-------------------------------------------------------------------------

>>>-------------------------------------------------------------------------
Exception in /home/travis/build/pandas-dev/pandas/doc/source/user_guide/io.rst at block ending on line 4230
Specify :okexcept: as an option in the ipython:: block to suppress this message
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-481-53221ddff055> in <module>
----> 1 store.select('wp')
~/build/pandas-dev/pandas/pandas/io/pytables.py in select(self, key, where, start, stop, columns, iterator, chunksize, auto_close, **kwargs)
    709         group = self.get_node(key)
    710         if group is None:
--> 711             raise KeyError('No object named {key} in the file'.format(key=key))
    712 
    713         # create the storer and axes
KeyError: 'No object named wp in the file'
<<<-------------------------------------------------------------------------
```

Reply:
Thanks. Removed.